boards/adi/max32655evkit/doc/index.rst

```diff
@@ -154,10 +154,73 @@ Debugging
 Please refer to the `Flashing`_ section and run the ``west debug`` command
 instead of ``west flash``.
 
+Dual Core Support
+*****************
+
+An experimental board configuration for the secondary RISC-V core can be found
+in
+:zephyr_file:`boards/adi/max32655evkit/max32655evkit_max32655_rv32_defconfig`.
```

Comment:
@kartben for comment, would suggest deleting section. Board configuration isn't a hwmv2 term